### ISSUE TITLE
Main navigation toggle button cannot receive focus, and collapsing menu loses focus

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/freedom#37]
When a window is resized smaller or zoomed in and navigation collapses under a toggle, the main navigation toggle button cannot be tabbed to/receive focus, thus deeming the entire navigation menu inaccessible. This is also the case if a user is accessing a site on mobile in portrait orientation (tested on Safari on iOS).

As what appears to be a consequence of this, if the collapsable menu is open and the "Close Menu" navigation control is then activated, focus is immediately lost once the menu closes. Also, the "Close Menu" navigation control seems to hide behind the main header bar on Chrome.

"Close Menu" button visible on Firefox:
<img width="846" alt="Screenshot of open navigation menu on Omeka S site on Firefox" src="https://github.com/user-attachments/assets/31699cc6-b6d0-4d94-a8e4-fe604ac6309f" />

"Close Menu" button hidden on Chrome:
<img width="848" alt="Screenshot of open navigation menu on Omeka S site on Chrome" src="https://github.com/user-attachments/assets/d97900fb-41c4-4f51-91ff-f1d281bd008f" />

**Comment by nelsonamaya82** [comment]
@aviannamiller this is fixed in the `master` branch, please double check and let me know if it works as expected.

**Comment by aviannamiller** [comment]
This is great, thank you. The nav control is now fully visible on Chrome, and the toggle can receive focus. Focus order looks good, too.
